cmd/config/main.go

```diff
@@ -383,23 +396,40 @@ func createMarketMap() (mmtypes.MarketMap, error) {
 			tickersToProviders[tickerStr] = mmtypes.Providers{Providers: providers}
 		}
 	}
-	if *usePathsOpt {
-		for ticker, paths := range TickerPaths {
-			optionalTickerPaths[ticker.String()] = paths
-		}
-	}
 
 	// Create a new market map from the provider to market map.
 	marketMap := mmtypes.MarketMap{
 		Tickers:   tickers,
 		Providers: tickersToProviders,
-		Paths:     optionalTickerPaths,
 	}
 
 	// Validate the market map.
 	if err := marketMap.ValidateBasic(); err != nil {
-		return mmtypes.MarketMap{}, fmt.Errorf("error validating the market map: %w", err)
+		return fmt.Errorf("error validating the market map: %w", err)
+	}
+
+	// Open the local market config file. This will overwrite any changes made to the
+	// local market config file.
+	f, err := os.Create(*marketCfgPath)
```

Comment:
Can we check if all flags were populated at the start of `main`?

Just so we can feel good about not having any `nil` dereferences.  It should be fine, but would be good to have the check in place

Reply:
I think `flags.Parse()` does this. If nothing exists it populates with the default value.